test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -9,9 +9,22 @@ Describe "Job Cmdlet Tests" -Tag "CI" {
         It "Start-Job produces a job object" {
```

Comment:
>        It "Start-Job produces a job object" {
>            $j.gettype().fullname | should be "System.Management.Automation.PSRemotingJob"
>         }
>       It "Start-Job can name the job" {
>            $j = Start-Job -ScriptBlock {6 * 7} -Name "My Job"
>            $j.Name | should be "My Job"
>        }
>       It "Get-Job retrieves a job object" {
>            (Get-Job -id $j.id).gettype().fullname | should be "System.Management.Automation.PSRemotingJob"
>        }

This may not strictly relevant, but for the 3 simple tests above, it seems too heavy to run `start-job -scriptblock { 1 + 1 }`, which essentially starts a process, before each test. Can we change the `BeforeEach` to
```
        BeforeEach {
            $j = start-job -scriptblock { 1 + 1 } -Name "My Job"
        }
```
and group those 3 tests into one?